test/api/v3/integration/chat/POST-chat.test.js

```diff
@@ -78,4 +79,31 @@ describe('POST /chat', () => {
     expect(message.message.id).to.exist;
     expect(memberWithNotification.newMessages[`${group._id}`]).to.exist;
   });
+
+  it('Returns an error when the user has been posting too many messages', async () => {
+    let spamError = {};
+
+    // this loop should only actually run 10 times if the test would fail, else it is cut short at first error
+    for (let i = 0; i < 5; i++) {
+      try {
```

Comment:
I think it would be better to not do this in a for loop, to better illustrate the threshold.